playbooks/roles/apticron/templates/apticron.conf.j2

```diff
@@ -0,0 +1,101 @@
+# apticron.conf
+#
+# set EMAIL to a space separated list of addresses which will be notified of
+# impending updates
+#
+EMAIL="{{ email_to }}"
+
+#
+# Set DIFF_ONLY to "1" to only output the difference of the current run
+# compared to the last run (ie. only new upgrades since the last run). If there
+# are no differences, no output/email will be generated. By default, apticron
+# will output everything that needs to be upgraded.
+#
+# DIFF_ONLY="1"
+
+#
+# Set LISTCHANGES_PROFILE if you would like apticron to invoke apt-listchanges
+# with the --profile option. You should add a corresponding profile to
+# /etc/apt/listchanges.conf
+#
+# LISTCHANGES_PROFILE="apticron"
+
+#
+# From hostname manpage: "Displays  all FQDNs of the machine. This option
+# enumerates all configured network addresses on all configured network inter‐
+# faces, and translates them to DNS domain names. Addresses that cannot be
+# translated (i.e. because they do not have an appro‐ priate  reverse DNS
+# entry) are skipped. Note that different addresses may resolve to the same
+# name, therefore the output may contain duplicate entries. Do not make any
+# assumptions about the order of the output."
+#
+# ALL_FQDNS="1"
+
+#
+# Set SYSTEM if you would like apticron to use something other than the output
+# of "hostname -f" for the system name in the mails it generates. This option
+# overrides the ALL_FQDNS above.
+#
+# XXX Consider using the ansible name.
+# SYSTEM="{{ inventory_hostname }}"
```

Comment:
@tartansandal This seems fine. I don't think it makes a huge amount of difference here? It probably could be even easier to leave it out and let apticron use `hostname -f`.

Note also "This option overrides the ALL_FQDNS above."

Reply:
All but the very first option (EMAIL) is commented out. The commented options are not the defaults, but suggestions. So `# ALL_FQDNS=1` is just a suggestion if you don't want the default of using `hostname -f`.  So the override is only relevant if that is commented out.

The `# SYSTEM` suggestion is just me being helpful to a future developer mucking around with this role. :smile: I'll make the comment more explicit, say, 
```
# XXX If you are having trouble with the system names returned by `hostname -f`,
# consider using the hostname used by the ansible inventory.
# SYSTEM="{{ inventory_hostname }}"
```

Reply:
Ah I missed that they were commented. :man_facepalming:  All good! :)